### DISCUSSION TITLE
Is global zk mandatory for geo replication?

### DISCUSSION THREAD

**Discussion by shulaoh** [apache/pulsar#22456] [Q&A] [answered]
Hi all,

I am just trying to setup 2 pulsar clusters with local zk for each pulsar cluster. now I want to configure geo replication between the 2 clusters, before I started working for this, I just want to know if the global zk is mandatory?

thanks for all.

**Comment by eolivelli** [comment, marked as answer]
It is not necessary

**Reply by shulaoh** [reply]
Thanks a lot, it works.
